README.md

```diff
@@ -1 +1,71 @@
-# seqqc-build-contam-db
+# Build a sourmash database to detect common sources of contamination in new sequencing data
```

Comment:
Maybe add a table of contents to the links for each header since it could get lengthy